### PR TITLE
Support false values in configuration file

### PR DESCRIPTION
**Pull request by ganchurin**
The purpose of this changes is to fix the issue: https://github.com/kubernetes-client/python/issues/954

### CODE REVIEW

File: config/kube_config_test.py [hunk 1]
```diff
@@ -564,6 +564,7 @@ class TestKubeConfigLoader(BaseTestCase):
                     "server": TEST_SSL_HOST,
                     "certificate-authority-data":
                         TEST_CERTIFICATE_AUTH_BASE64,
+                    "insecure-skip-tls-verify": False,
```
Comment (roycaihw) [inline config/kube_config_test.py]: s/False/"false" in consistent with the test case below

Comment (ganchurin) [reply config/kube_config_test.py]: @roycaihw Thanks for the suggestion! I've just tried to replace `False` with `"false"` and faced an issue - the unit test fails. I think the cause is in how "insecure-skip-tls-verify" property is handled in `kube_config.py` (https://github.com/kubernetes-client/python-base/blob/master/config/kube_config.py#L443). The code by the link casts a string to a boolean. Both `"true"` and `"false"` are casted to `True`. What do you think if I replace `"true"` with `True` in the older test?

Comment (roycaihw) [reply config/kube_config_test.py]: `"true"` and `"false"` are the original values in kubeconfig yaml files (e.g. `insecure-skip-tls-verify: false`). The test cases are testing values stored in the `KubeConfigLoader` object. 

Could you verify what's the behavior in the [yaml loader](https://github.com/kubernetes-client/python-base/blob/af42f24d726dc4a8e5aaca63ee30ccae93d64f1b/config/kube_config.py#L580)? Does it load `"false"` into a boolean `False`, or does it keep the string value `"false"`?

Comment (ganchurin) [reply config/kube_config_test.py]: I verified the behavior on the configuration file which has the following lines:
```
clusters:
- cluster:
    insecure-skip-tls-verify: false
```

The following code snippet was used for the verification:
```
import yaml

with open(<PATH_TO_CONFIG>) as f:
    config = yaml.safe_load(f)
    print config (**breakpoint was set here**)
```

Debug showed the following value for the property:
`'insecure-skip-tls-verify' (4523129968) = {bool} False`

Comment (roycaihw) [reply config/kube_config_test.py]: Thanks. Let's use `True` and `False` in the test cases then

**Comment by roycaihw** [comment]
/lgtm
/approve

Thanks @ganchurin!

**Comment by ganchurin** [comment]
@roycaihw Could you please help me a bit? Some tests have failed after my update to the branch. I'm not sure that they relate to the scope of work. And I can't restart test run. Thanks in advance!

**Comment by micw523** [comment]
/ok-to-test
???? What’s happening here? I didn’t remember we required ok to test

**Comment by micw523** [comment]
The Travis CI is showing a ton of 404 and other errors. Try looking at the log - it is available.

**Comment by ganchurin** [comment]
@micw523 Thank you! I've taken a look.

I've noticed that the same tests fail in the latest build of "kubernetes-client/python" repository: https://travis-ci.org/kubernetes-client/python/builds/587650221

As for my pull request, previously the pull request had passed successfully. The single difference between successful and failed attempts is one line in unit tests that works fine for me locally. Thus I tend thinking that the issue is in "kubernetes-client/python" repository.

**Comment by micw523** [comment]
The most recent build in k8s-python passed; I'm restarting your build to see if the test fail is a flake.

**Comment by ganchurin** [comment]
@micw523 Sorry if I'm wrong - I was looking for builds here: https://travis-ci.org/kubernetes-client/python/builds. And the latest build is red.

**Comment by roycaihw** [comment]
Test failure tracked in https://github.com/kubernetes-client/python/issues/963. Let's fix the CI first and then merge this PR

**Pull request merged**